control/utils.go

```diff
@@ -128,6 +128,10 @@ func SetForwarding(ifname string, val string) {
 	_ = setForwarding(ifname, consts.IpVersionStr_6, val)
 }
 
+func SetAcceptLocal(ifname, val string) error {
+	return os.WriteFile(fmt.Sprintf("/proc/sys/net/ipv4/conf/%s/accept_local", ifname), []byte(val), 0644)
```

Comment:
Does ipv6 need it?

Reply:
There is no `net.ipv6.conf.$wan.accept_local` sysctl entry. This is actually normal for kernel, another case is `tcp_early_demux`: only `net.ipv4.tcp_early_demux` exists but not for `net.ipv6.tcp_early_demux`.